ome_zarr/reader.py

```diff
@@ -47,22 +50,64 @@ def __init__(
         if OMERO.matches(zarr):
             self.specs.append(OMERO(self))
 
+    @property
+    def visible(self) -> bool:
+        """True if this node should be displayed by default.
+
+        An invisible node may have been requested by the instrument, by the
+        user, or by the ome_zarr library after determining that this node
+        is lower priority, e.g. to prevent too many nodes from being shown
+        at once.
+        """
+        return self.__visible
+
+    @visible.setter
+    def visible(self, visibility: bool) -> bool:
+        """
+        Set the visibility for this node, returning the previous value.
+
+        A change of the visibility will propagate to all subnodes.
+        """
+        old = self.__visible
+        if old != visibility:
+            self.__visible = visibility
+            for node in self.pre_nodes + self.post_nodes:
+                node.visible = visibility
+        return old
+
     def load(self, spec_type: Type["Spec"]) -> Optional["Spec"]:
         for spec in self.specs:
             if isinstance(spec, spec_type):
                 return spec
         return None
 
-    def add(self, zarr: BaseZarrLocation, prepend: bool = False,) -> "Optional[Node]":
-        """Create a child node if this location has not yet been seen; otherwise return
-        None."""
+    def add(
+        self,
+        zarr: BaseZarrLocation,
+        prepend: bool = False,
+        visibility: Optional[bool] = None,
+    ) -> "Optional[Node]":
+        """Create a child node if this location has not yet been seen.
+
+        Returns None if the node has already been processed.
+
+        By setting prepend, the addition will be considered as higher priority
+        node which will likely be turned into a lower layer for display.
+
+        By unsetting visible, the node (and in turn the layer) can be
+        deactivated for initial display. By default, the value of the current
```

Comment:
Slightly confused: "by default" suggests a means of override but I am not seeing what that is.

Reply:
Passing a value would override the None. I'm pushing a commit to rework the wording. Let me know if you think it's better.